.github/workflows/tests.yaml

```diff
@@ -0,0 +1,39 @@
+name: Tests
+
+on:
```

Comment:
let's also run them on `main` pushes

Reply:
Actually, this job is dropped by the follow-up PR as we're switching to the shared "run_tests" job: https://github.com/canonical/multus-rock/pull/6/files#diff-7b2a967de7060a238e49a57908f83e860b8cb53a0b0991c3d0d275857c1014d4R21-R26